index.html

```diff
@@ -13,16 +13,21 @@
 
 <body>
   <main class="toolbar">
-    <div class="nav-menu">
+    <nav class="nav-menu">
       <h4 class="welcome"><a href="#">Welcome</a></h4>
```

Comment:
- [ ] As I can see here the nav is still there the `welcome` sice it is not in the Figma template Kindly remove it to match the Figma as much as possible ✅✅ 


- Your design
![image](https://user-images.githubusercontent.com/58553711/179644871-ee0cf8d1-4529-41e6-b1d7-3b293147f3ad.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179644955-f5401e4b-590f-41fa-9542-17e3a394d388.png)